resources/assets/less/basics/colours.less

```diff
@@ -55,7 +55,7 @@
 .c-bluedark {
   color: @muhitBlueDark;
 }
-.c-inprogress {
+.c-in-progress {
```

Comment:
@aniluyg what is the reason of the name change here? Doesn't seem to do or solve anything; I would suggest removing this?

Reply:
I need to use issue status to set required css class name dynamically in html elements. To achieve this i've used getIssueStatus() function into show.blade.php. You can see similar usage in issues-list.blade.php. getIssueStatus() check for "in-progress" status and returns "in-progress" because this status name saved in issues table.  If we use inprogress for style  we need to use new function. I checked that c-inprogress is not used anywhere. So that i renamed it with c-in-progress css class. If we undo this change icons color for inprogress issues will not seem as we want.

Reply:
Okay I see; creates a slight inconsistency w other class names, but doesn't seem to be a problem.